<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color:white;
}

QLabel{
	font-family: Arial;
	font-size: 14px;
}

QPushButton{
	background-color: #ed5684;
	color: white;
	min-height:30px;
	border-radius: 20px;
}

QLineEdit{
	border: 1px solid #d0d0d0;
	border-radius: 20px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item alignment="Qt::AlignVCenter">
    <widget class="QFrame" name="asdasdaa">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:26px;
color: #ed5684;</string>
        </property>
        <property name="text">
         <string>Вхід</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Номер телефону</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="email_line">
            <property name="styleSheet">
             <string notr="true">min-height:30px;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Пароль</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="password_line">
            <property name="styleSheet">
             <string notr="true">min-height:30px;</string>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="redirect_button_register">
          <property name="text">
           <string>Перейти до реєстрації</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="login_button">
          <property name="text">
           <string>Увійти</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
